ftdi-gpio.c

```diff
@@ -38,11 +38,12 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include "cdba-server.h"
 #include "ftdi-gpio.h"
 
-#include "ftdi.h"
+#include <libftdi1/ftdi.h>
```

Comment:
I think adding the include directory to CFLAGS is preferable to having the code include `libftddi1/ftdi.h`.